### DISCUSSION TITLE
Is it possible to reduce the frame rate to 10fps

### DISCUSSION THREAD

**Discussion by foilandwater** [owncast/owncast#3782] [Q&A] [answered]
Hi,

Under configuration -> video, you can choose the profile but I noticed that the lowest frames per second is 24fps. Is it possible somehow to go lower? I'm receiving a 1280x720 10fps stream so being forced to stream at 24fps is a waste. By halving the frame rate I can gain a better quality whilst using the same bitrate.

Appreciate any advice, thanks Sean.

**Comment by gabek** [comment, marked as answer]
I've never tried it that low, so honestly, I don't know if you might run into any problems or not. In theory, it should be fine. You can use the API to set any custom framerate you want. There's discussion about how to do it at https://github.com/owncast/owncast/issues/1462, but it's not something that's officially supported currently.

**Reply by foilandwater** [reply]
Thanks a lot for the reply. Unfortunately I do not understand enough to know how to access the API. I read the thread you referred to and did some other digging but I just wasn't able to figure it out. I got as far as opening devtools within chrome whilst logged into the downcast admin interface and could not get further than that.
